core-services/image-mirroring/integr8ly/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+- maleck13
+- mikenairn
+options: {}
```

Comment:
@laurafitzgerald Can you remove Craig and add yourself, Paul and Steven who are all members of the openshift org.

Reply:
@mikenairn I see Craig listed in other owner files. Should we remove him from those?